README.md

```diff
@@ -128,6 +128,18 @@ fastify.register(jwt, {
     callback(null, 'supersecret')
   }
 })
+// secret as a promise
+fastify.register(jwt, {
+  secret: async function (request, reply) {
```

Comment:
not need for this to be async if you're not awaiting anything. alternatively you could keep this async and show something that needs awaiting in the body of the function instead

Reply:
Removed async